ruby_test.rb

```diff
@@ -0,0 +1,8 @@
+puts "one"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.